src/main/java/com/parasoft/demoapp/service/ParasoftJDBCProxyService.java

```diff
@@ -137,9 +137,11 @@ public void validateVirtualizeServerUrl(String virtualizeServerUrl) throws Virtu
             }
         } catch (Exception e) {
             e.printStackTrace();
+            // parasoft-begin-suppress EXCEPT.CTE-3 "expected"
             throw new VirtualizeServerUrlException(
                     MessageFormat.format(GlobalPreferencesMessages.INVALIDATE_PARASOFT_VIRTUALIZE_SERVER_URL,
                                             virtualizeServerUrl,  e.getMessage()));
+            // parasoft-end-suppress EXCEPT.CTE-3
```

Comment:
Please use the other constructor of VirtualizeServerUrlException and pass in the exception instead of suppress it.